_rules/image-no-text-0va7u6.md

```diff
@@ -180,29 +182,40 @@ This image resource referenced by the `img` element contains text for which the
 
 #### Failed Example 2
 
-This image resource referenced by the `input` element in the [Image Button][] contains text for which the particular presentation is not essential.
+This image resource referenced by the `input` element in the [Image Button][] contains text for which the particular presentation is not [essential](#0va7u6:essential).
 
 ```html
 <input type="image" src="/test-assets/0va7u6/button.jpg" alt="Press me" />
 ```
 
 #### Failed Example 3
 
-This image resource referenced by the `background-image` property of the `div` element contains text for which the particular presentation is not essential.
+This image resource referenced by the `background-image` property of the `div` element contains text for which the particular presentation is not [essential](#0va7u6:essential).
 
 ```html
 <div style="background-image: url(/test-assets/0va7u6/textimage.jpg); width: 500px; height: 200px;" />
 ```
 
 #### Failed Example 4
 
-This image resource referenced by the `img` element contains text that provides redundant information, but it still is information, therefore it is not [purely decorative][].
+This image resource referenced by the `img` element contains text that provides redundant information, but it still is information, therefore it is not [decorative](#0va7u6:decorative).
 
 ```html
 <img src="/test-assets/0va7u6/welcome.png" alt="" />
 <p>Welcome to our website</p>
 ```
 
+#### Failed Example 5
+
+This `img` elements loads an SVG with text as an image resource. Because the SVG is loaded as an image resource, instead of being embedded in HTML the text cannot be selected or customized.
```

Comment:
```suggestion
This `img` element loads an SVG with text as an image resource. Because the SVG is loaded as an image resource, instead of being embedded in HTML the text cannot be selected or customized.
```